### ISSUE TITLE
key rebinding

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#9]
users should be able to choose whatever key combinations they like for the various functions that are currently hardcoded in individual widget classes

**Comment by HeladoDeBrownie** [comment]
relatedly, need to test with at least one non-us qwerty keyboard layout, such as dvorak
